src/crepe/api/GameObject.h

```diff
@@ -3,6 +3,8 @@
 #include <cstdint>
 #include <string>
 
+#include "Transform.h"
```

Comment:
Vector2 moet in een apart bestand, dus deze include moet veranderd worden. Daarnaast is een incomplete class declaratie van Vector2 hier voldoende.
Let op: Deze file hoort dan wellicht niet bij jouw PR, maar GameObject is inmiddels helemaal volgens de code standaard en dat houden we ook graag zo qua mij betreft.